packages/legacy/core/App/index.ts

```diff
@@ -141,6 +141,7 @@ export {
   AttachTourStep,
   TourBox,
   defaultConfig,
+  defaultHistoryEventsLogger,
```

Comment:
Can you also export `HistoryEventsLoggerConfig`

Reply:
@MosCD3 done!